users/views.py

```diff
@@ -103,6 +105,23 @@ def index(request):
     )
 
 
+def change_password(request):
```

Comment:
You could use all auth view for changing password https://django-allauth.readthedocs.io/en/latest/views.html#password-management.

Reply:
Done